src/main/java/io/github/thebusybiscuit/sensibletoolbox/listeners/ElevatorListener.java

```diff
@@ -65,4 +67,10 @@ public void onPlayerSneak(PlayerToggleSneakEvent event) {
         }
     }
 
+    private boolean isOnGround(@Nonnull Player player) {
+        Location location = player.getLocation();
+        Block currentBlock = location.getBlock();
+        return !currentBlock.getRelative(BlockFace.DOWN).isEmpty() && Math.abs(location.getY() - location.getBlockY()) < 0.05;
```

Comment:
The `Math.abs()` is redundant here, since you already determined the direction of the movement to be upwards in line 28. So you can omit this call.